auditlog/models/rule.py

```diff
@@ -343,6 +358,16 @@ def read(self, fields=None, load="_classic_read", **kwargs):
                 return result
             self = self.with_context(auditlog_disabled=True)
             rule_model = self.env["auditlog.rule"]
+            rule = rule_model.sudo().search([("model_id.model", "=", self._name)])
+            if self.env.user in rule.mapped("users_to_exclude"):
+                return result
+            fields_to_exclude_match = [
+                d in rule.fields_to_exclude.mapped("name") for d in fields
```

Comment:
instead of looping on all of them I would do a simple loop and break on the first that match

Reply:
also this block could be moved to its on method

Reply:
Done with break in loop